### ISSUE TITLE
Create a roslyn-based installation script for modules

### ISSUE THREAD

**Issue by derjabkin** [eXpandFramework/PocketXAF#6]
When installing a packages with a module, module should be automatically added to the target module or application

**Comment by apobekiaris** [comment]
this I am very interested to see how is done, I guess I can apply it to other cases such as the registration part of the expression evaluator

https://github.com/expand/eXpand/issues/140

**Comment by apobekiaris** [comment]
reference the commit please when u done

**Comment by derjabkin** [comment]
Yes, will do. This will take a while. First I would integrated something, that has a module
